data/base-consumables.yaml

```diff
@@ -80,6 +80,105 @@ Shard:
     size: 1
     stackable: false
     quantity: 1
+  ### Runestones ###
+  ec_rune:
```

Comment:
Thoughts on naming the runes with their full spell name, or including the full spell name as a property or as a comment? I don't have the spell acronyms memorized so having the spell names listed somewhere to see at a glance here would be helpful.

Reply:
Done. I could now use the restock logic in the purchase def and key in only on the one we want instead of running all the restock items.